### ISSUE TITLE
Expose the K8s cilium network policy UUID in the node status

### ISSUE THREAD

**Issue by ashwinp** [cilium/cilium#4153]
### Title

Expose the K8s cilium network policy UUID in the node status

### Description of requested functionality

Today, if you do a `kubectl get cnp -o json`, you'll observe that we expose the status of the policy on each node. As an example, the output on the single node minikube cluster looks something like:
```
    200             "status": {
    201                 "nodes": {
    202                     "minikube": {
    203                         "enforcing": true,
    204                         "lastUpdated": "2018-05-17T00:04:36.958102189Z",
    205                         "localPolicyRevision": 20,
    206                         "ok": true
    207                     }
    208                 }
    209             }
```
The aforementioned status does tell you the fact that a policy is actually being enforced on the node. However, it does not tell you which policy is currently being enforced.

I see that the name of the policy is available in the output of `kubectl get cep`. However, this does not help identify a unique version of the policy, as I can update the policy and apply it with the same name.

It would be useful if the `kubectl get cnp` status also includes a unique identifier of the cilium network policy. Taking a cursory look at the `kubectl get cnp` output suggests that the `uid` field is a good candidate for this purpose.

To summarize, the proposal is to include a policy `uid` in the output of `kubectl get cnp` status field. As an example:
```
    200             "status": {
    201                 "nodes": {
    202                     "minikube": {
    203                         "enforcing": true,
    NEW                        "uid": "e0fad9a7-5965-11e8-b4cb-080027d537e1"
    204                         "lastUpdated": "2018-05-17T00:04:36.958102189Z",
    205                         "localPolicyRevision": 20,
    206                         "ok": true
    207                     }
    208                 }
    209             }
```
